### DISCUSSION TITLE
流量平白无故的消失了？？

### DISCUSSION THREAD

**Discussion by tianqo** [natfrp/wiki#126] [使用问题] [answered]
前几个月攒了一百多G的流量，今天醒来签个到怎么就剩25了？？
![图片](https://user-images.githubusercontent.com/68796895/182069522-e4faae47-30bb-48d7-b27a-988686428cb2.png)

这没道理啊，最近一次使用也只是上个星期和朋友玩了四五小时的我的世界，不会这要消耗四十多个G的流量吧？
![图片](https://user-images.githubusercontent.com/68796895/182069700-cb0da2c6-e5c0-4574-b461-91a5c38c477c.png)

时间都已经被冲掉了，想看都看不到了，但至少也得给个解释吧。

**Comment by SanaeAkahane** [comment]
Minecraft Java版 消耗流量快 挺正常的
亦可参考 [帮助文档](https://doc.natfrp.com/#/bestpractice/security?id=frpc-%e8%ae%bf%e9%97%ae%e8%ae%a4%e8%af%81) ，其中写出了流量凭空消失的可能解决方案。

**Comment by KochiyaPatchy** [comment]
我也是耶
![Screenshot_20220803_180004_com huawei browser_edit_162739828237667](https://user-images.githubusercontent.com/110249844/182583718-4aa7f9b8-fba0-4045-86a6-9cdcafd25adb.jpg)

[a comment by an automated account is not shown]
